ilastik/__init__.py

```diff
@@ -29,10 +29,10 @@
 this_file = os.path.realpath( this_file )
 ilastik_package_dir = os.path.dirname(this_file)
 ilastik_repo_dir = os.path.dirname(ilastik_package_dir)
-submodule_dir = os.path.join( ilastik_repo_dir, 'submodules' )
+submodule_dir = os.path.dirname(ilastik_repo_dir)
```

Comment:
Yes, relative imports were otherwise causing problem. The submodule_dir was having incorrect dir path, and the same were "exposed". 
Python3 was unable to find the needed modules during imports.